men_team_1/db/migrate/20181109011844_add_topic_to_usertag.rb

```diff
@@ -0,0 +1,5 @@
+class AddTopicToUsertag < ActiveRecord::Migration[5.2]
+  def change
+    add_reference :usertags, :topic_id, foreign_key: true
```

Comment:
Reference names in migrations should never end with `_id`. Rails appends that to the column name for you automagically. Should be `:usertags, :topic`.